### ISSUE TITLE
Temperature icons in the tray get duplicated after some time

### ISSUE THREAD

**Issue by Taudris** [Rem0o/FanControl.Releases#1651]
**Describe the bug**

I have two sensors configured as tray icons. At some point, they become frozen (circled in red) and new icons are added that work as expected.

Other facts:
* Exiting FanControl causes the icons to disappear on their own; I don't have to hover my mouse over them to get them to disappear as one often must with "forgotten" tray icons.
* Correct tooltips appear when hovering the icons.
* The right click menu does not appear on the frozen icons.
* I don't know what triggers this bug.
* I regularly sleep and wake this computer.
* I have yet to see more than one extra set of icons, but that could be because I usually restart FanControl when I see this occur. (I'm now going to try leaving it running as-is to see if I get more.)

![image](https://user-images.githubusercontent.com/1720131/227467322-791b61d4-6ba7-4f45-903f-f00517c2a143.png)

**Is there a log.txt file next to FanControl.exe with recent date entries?**
Last entry was prior to the last system reboot.

**Relevant hardware specs and setup**
* Windows 11 Pro 22H2.
* 5900X, Gigabyte B550M AORUS PRO-P, 4090. All sensors and fan controls work as expected.

**Comment by Taudris** [comment]
I now have 3 CPU temp icons (1 alive, 2 dead) and 2 GPU temp icons (1 alive, 1 dead).

I also just noticed that dragging to change the tray icon order doesn't always work correctly; the icons often end up in an unexpected order.

**Comment by Taudris** [comment]
On V153, the icons no longer seem to be duplicating, but they do stop updating and the right-click menu stops working. The fan control aspect does seem to be working, but I can't interact with it until I terminate the process and restart it.

**Comment by Taudris** [comment]
nvm, still duplicating icons with the same symptoms. So basically, sometimes the icons die, and there are *usually* new working icons added, but not always.

(@Rem0o just in case it helps)

**Comment by rwohleb** [comment]
I also get the duplicating icons, and it seems to only happen after the machine goes to sleep and wakes up again. Double clicking on the icons that are getting refreshed properly brings up the UI, as expected. The old/dupe icons that aren't getting refreshed do nothing when you double click on them.

**Comment by Rem0o** [comment]
Still an issue?

**Comment by rwohleb** [comment]
Yes, just had dupe icons appear when my desktop woke from sleep this morning.

**Comment by Taudris** [comment]
Yup, still an issue. Definitely seems to be in connection with sleep/resume.

I just confirmed that the icons being dead does not mean that FanControl has stopped working,  though it does make it impossible to open the UI. In HWiNFO, I can see fan RPMs responding to system load as normal.

**Comment by Taudris** [comment]
Found this again by accident and realized I haven't seen it in a while. Closing.
